File: Client_Root/Client/Assets/Scripts/MasterData/Item.cs
using System.Collections.Generic;

namespace MasterData
{
    public class Item : IMasterData
    {
        public string m_strName;
        public string m_strClassName;
        public List<int> m_listBehaviorID = new List<int>();
        public float m_fSize;
        public string m_strModelResName;

        public override void SetData(List<string> data)
        {
            Util.Convert(data[0], ref m_nID);
            m_strName = data[1];
            m_strClassName = data[2];
            Util.Parse(data[3], ',', m_listBehaviorID);
            Util.Convert(data[4], ref m_fSize);
            m_strModelResName = data[5];
        }
    }
}

File: Server_Root/Server/RoomServer/MasterData/Item.cpp
#include "stdafx.h"
#include "Item.h"
#include "../Util.h"

namespace MasterData
{
	Item::Item()
	{
	}

	Item::~Item()
	{
	}

	void Item::SetData(vector<string> data)
	{
		m_nID = atoi(data[0].c_str());
		m_strName = data[1];
		m_strClassName = data[2];
		Util::Parse(data[3], ',', &m_vecBehaviorID);
		m_fSize = atof(data[4].c_str());
		m_fLifespan = atof(data[6].c_str());
		m_fHeight = atof(data[7].c_str());
		m_fDefault_Y = atof(data[8].c_str());
		m_nDefaultBehaviorID = atoi(data[9].c_str());
		m_strEffectType = data[10];
		m_nEffectParam = atoi(data[11].c_str());
	}
}